cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -820,9 +801,10 @@ func newServer(conf rootConfig) (*Server, error) {
 	_, _ = maxprocs.Set()
 
 	// Set the sampling rate of Go's block profiler: https://github.com/DataDog/go-profiler-notes/blob/main/guide/README.md#block-profiler.
-	// The block profiler is disabled by default.
+	// The block profiler is disabled by default and should be enabled with care in production
 	runtime.SetBlockProfileRate(conf.blockProfileRate)
 
+	profiler.Init("zoekt-sourcegraph-indexserver", zoekt.Version, conf.blockProfileRate)
```

Comment:
can you just make it so that we call runtime.SetBlockProfileRate inside of profiler.Init? That way webserver also gets to benefit from it.

cc @ggilmore do we actually use this anymore? Should we just get rid of it?

Reply:
Are you talking about the block profiler itself? We don't run it continuously, but we have the machinery to enable it by setting an environment variable (this would we be useful while we're debugging something).